documentation/content/en/books/handbook/desktop/_index.adoc

```diff
@@ -74,7 +74,7 @@ A desktop environment can range from a simple window manager to a complete suite
 
 | KDE Plasma
 | GPL 2.0 or later
-| x11/kde5
+| x11/kde6
```

Comment:
```suggestion
| x11/kde
```

Reply:
Thanks for the suggestion, I've now addressed it